### PR TITLE
Update frontend + .NET 7

### PR DESCRIPTION
**Pull request by Saibamen**

### CODE REVIEW

**Comment by Saibamen** [comment]
@gdl-reif: Important: `"type": "module"` was added to `package.json`

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Development Template for IDAS micro apps
 
-**Goal**: this template enables developers to quickly set up a web application with the following 
+**Goal**: this template enables developers to quickly set up a web application with the following
 features included:
 
 * Authentication against the IDAS backend (providing a JWT token)
 * Sample Svelte frontend code
-* App-local .NET6 backend 
+* App-local .NET6 backend
```
Comment (gdl-reif) [inline README.md:8]: NET7

**Review by gdl-reif — APPROVED** [review]

**Comment by Saibamen** [comment]
AppToken GUID validator:

image

File: README.md (hunk 1 above)
Comment (Saibamen) [reply README.md]: Done

**Review by gdl-reif — APPROVED** [review]

**Pull request merged**